docker-build.sh

```diff
@@ -26,9 +27,24 @@ do
     docker push $REPO/$PREFIX-$IMAGE:$NEW_TAG
 done
 
-for IMAGE in 'dask' 'mlrun-api' 'test'
+# need some python 3.6 for (legacy) package consistency 
+for IMAGE in 'base' 'models' 'models-gpu'
```

Comment:
@yjb-ds would be nice to have a partial build option (only some containers or split to 2-3 groups), it usually kills me VM disk/mem space to build all, or sometimes i need just 1-2 images and need to wait 40min for all the build

Reply:
this will be dealt with in next update